### DISCUSSION TITLE
Reed-Solomon - support for multiple tiles

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#309] [Q&A] [answered]
Hi team,

Can you please explain what the "support for multiple tiles" means in regards to RS FEC.

Thanks,
Alan

**Comment by MartinPulec** [comment, marked as answer]
UltraGrid has a multi-tile feature that processes multiple video tiles independently. It has been used in history for [4K](https://github.com/CESNET/UltraGrid/wiki/4K-Video-Settings), where multiple devices were used to capture individual tiles.

Currently, a more relevant use is perhaps for full-frame [stereoscopic 3D](https://github.com/CESNET/UltraGrid/wiki/Stereoscopic-3D-Video-Settings) video.

**Comment by alatteri** [comment]
thank you.
